build.gradle

```diff
@@ -27,7 +27,7 @@ buildscript {
 }
 
 plugins {
-    id 'com.github.johnrengelman.shadow' version '1.2.3'
+    id 'com.github.johnrengelman.shadow' version '2.0.0'
```

Comment:
I looked through the breaking changes in the changelog on http://imperceptiblethoughts.com/shadow/#change_log - don't see any problem here, though if anyone wants to give a sanity check it would be appreciated.